File: pset1/mario_less/mario.c
int main(void){
    //declaring variables
    int n, c = 1;
    int x = 0;
    //getting from de user a height between 1 and 8
    do{
        printf("type the pyramid height between 1 and 8\n");
        scanf("%i",&n);
    } while(n < 1 || n > 8);
    while(c <= n){
        //printing the space between the start of the line and the # of that line
        for (int j = 0; j < (n - c); j++){
            printf(" ");
        }
    //printing the # of the pyramides
        for (int i = 0; i < c; i++){
            printf("#");
        }
        c++;
    }
}

File: pset6/mario_hard/mario.py
from cs50 import get_int

height = 0
#getting the height from the user, the height must be between 1 and 8
while height < 1 or height > 8:
    height = get_int("Height: ")
    if height < 1 or height > 8:
        print("height vallue must be between 1 and 8")
#loop to iterate through all lines
for i in range (height):
    print(" " * (height - 1 - i), end = "") #printing spaces before 1st pyramide
    print("#" * (i + 1), end = "  ") #printing hashes from 1st pyramide/space at the end
    print("#" * (i + 1))  #printing hashes from 2nd pyramide


